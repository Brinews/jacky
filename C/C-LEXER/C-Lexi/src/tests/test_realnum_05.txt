1.00
abc
