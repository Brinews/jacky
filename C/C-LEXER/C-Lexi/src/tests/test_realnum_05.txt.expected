{1.00 , REALNUM , 1}
{abc , ID , 2}
{ , END_OF_FILE , 4}
